src/driver/amdxdna/amdxdna_ctx.h

```diff
@@ -111,6 +111,15 @@ struct amdxdna_hwctx {
 #define HWCTX_STAT_STOP  2
 	u32				status;
 	u32				old_status;
+	/*
+	 * Sequence number for next job; As this is start from 0,
+	 * it can be used as the submitted job counter.
+	 */
+	u64				seq;
```

Comment:
"seq" -> "submitted" and remove the comment

Reply:
fixed